scripts/test/system.sh

```diff
@@ -187,7 +188,50 @@ _subctl cloud prepare generic --context cluster1
 # Deprecated variant
 _subctl cloud prepare generic --kubecontext cluster1 && exit 1
 
+# Rename old broker-info.subm to be able to compare it with the recovered broker-info.sum
+echo "Renaming broker-info.sum to broker-info.subm.orig"
+mv "${DAPPER_SOURCE}"/output/broker-info.subm "${DAPPER_SOURCE}"/output/broker-info.subm.orig
+
+# all blues
+# Test subctl recover-broker-info invocations which will always fail (no matter the broker and Submariner installations and the current
+# context set) with broker not found error
+_subctl recover-broker-info --namespace non-existent && exit 1 #column 2
+_subctl recover-broker-info --namespace submariner-k8s-broker --brokercontext cluster2 && exit 1 #column 4
+_subctl recover-broker-info --context cluster2 && exit 1 #column 7
+_subctl recover-broker-info && exit 1 #5:1 - 8:1
+
+# Test subctl recover-broker-info invocations with both Broker and Submariner installed  # remaining row 8
+_subctl recover-broker-info --namespace submariner-k8s-broker --brokercontext cluster1
+cmp broker-info.subm.orig broker-info.subm
+_subctl recover-broker-info --namespace submariner-k8s-broker --brokercontext cluster1 --context cluster2
+cmp broker-info.subm.orig broker-info.subm
+
+if [ "$(kubectl config current-context)" == "cluster1" ]; then # finishes row 4
+  _subctl recover-broker-info
+  cmp broker-info.subm.orig broker-info.subm
+fi
+
 # Test subctl uninstall invocations
+_subctl uninstall -y --kubeconfig "${KUBECONFIGS_DIR}"/kind-config-cluster1 && exit 1
+
+# Test subctl recover-broker-info invocations with Submariner not installed on Broker cluster (Row 2, 6)
+_subctl recover-broker-info --namespace submariner-k8s-broker --brokercontext cluster1 --context cluster2 #2:6, 6:6
+cmp broker-info.subm.orig broker-info.subm
+
+# All the other commands fail
+_subctl recover-broker-info && exit 1
```

Comment:
We've already tested no args above.

Reply:
No args is effectively the same as `--context` and `--namespace` so it's redundant. Plus, as pointed out earlier, it's non-deterministic which context is current. Also on L209 above you do a context check but here you don't - it's inconsistent.